services/cron.ts

```diff
@@ -0,0 +1,32 @@
+const cron = require('node-cron');
+const Routine = require('../models/routine.model');
+
+// run a cron job every minute
+export const checkEveryMinute= cron.schedule('* * * * *', async () => {
```

Comment:
Are you sure this runs crobjob every one minute?

Reply:
yes